### PR TITLE
testing/e2tools: modernize, fix license

### PR DESCRIPTION
**Pull request by maxice8**

### CODE REVIEW

File: testing/e2tools/APKBUILD [hunk 1]
```diff
@@ -2,9 +2,9 @@
 # Maintainer: Daniel Santana <daniel@santana.tech>
 pkgname=e2tools
 pkgver=0.0.16
-pkgrel=0
+pkgrel=1
 pkgdesc="Utilities for manipulating files in an ext2/ext3 filesystem"
-url="http://home.earthlink.net/~k_sheff/sw/e2tools/"
+url="https://manpages.ubuntu.com/manpages/bionic/man7/e2tools.7.html" # original hoempage is dead
```
Comment (HRio) [inline testing/e2tools/APKBUILD]: does the source link really work then?

Comment (HRio) [reply testing/e2tools/APKBUILD]: is it the same e2tools as this? 

http://e2tools.github.io/
https://github.com/e2tools/e2tools/releases

Comment (maxice8) [reply testing/e2tools/APKBUILD]: Yes, the source link works but the homepage no

**Comment by maxice8** [comment]
On Fri, Jun 07, 2019 at 03:09:04AM -0700, Henrik Riomar wrote:
> HRio commented on this pull request.
> 
> 
> 
> >  pkgdesc="Utilities for manipulating files in an ext2/ext3 filesystem"
> -url="http://home.earthlink.net/~k_sheff/sw/e2tools/"
> +url="https://manpages.ubuntu.com/manpages/bionic/man7/e2tools.7.html" # original hoempage is dead
> 
> is it the same e2tools as this? 
> 
> http://e2tools.github.io/
> https://github.com/e2tools/e2tools/releases
> 
> -- 
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/alpinelinux/aports/pull/8522#discussion_r291531148

Seems to be

File: testing/e2tools/APKBUILD (hunk 1 above)
Comment (HRio) [reply testing/e2tools/APKBUILD]: OK, source link does not work for me, I get 404

**Comment by maxice8** [comment]
Updated the URLs

**Pull request closed without merging**